lib/PrefsFeed.jsm

```diff
@@ -46,8 +46,9 @@ this.PrefsFeed = class PrefsFeed {
   }
 
   onPrefChanged(name, value) {
-    if (this._prefMap.has(name)) {
-      this.store.dispatch(ac.BroadcastToContent({type: at.PREF_CHANGED, data: {name, value}}));
+    const prefItem = this._prefMap.get(name);
+    if (prefItem) {
```

Comment:
this broke pref changes to prefs that were set to falsy values. Fix in pull #4803